content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -118,30 +124,30 @@ to manage the lifecycle of Istio.
 
 ## Uninstall
 
-* For option 1, uninstall using `kubectl`:
+- For option 1, uninstall using `kubectl`:
 
     {{< text bash >}}
     $ kubectl delete -f $HOME/istio.yaml
     {{< /text >}}
 
-* For option 2, uninstall using Helm:
+- For option 2, uninstall using Helm:
 
-    {{< tip >}}
-    Uninstalling this chart does not delete Istio's registered CRDs. Istio by design expects
-    CRDs to leak into the Kubernetes environment. As CRDs contain all runtime configuration
-    data in custom resources the Istio designers feel it is better to explicitly delete this
-    configuration rather then unexpectedly lose it.
-    {{< /tip >}}
+    {{< warning >}}
+    Uninstalling this chart does not delete Istio's registered CRDs. Istio, by design, expects
+    CRDs to leak into the Kubernetes environment. As CRDs contain all the runtime configuration
+    data needed to configure Istio. Because of this, we consider it better for operators to
+    explicitly delete the runtime configuration data rather than unexpectedly lose it.
+    {{< /warning >}}
 
     {{< text bash >}}
     $ helm delete --purge istio
     $ helm delete --purge istio-init
     {{< /text >}}
 
-* If desired, run the following command to delete all CRDs:
+- If desired, run the following command to delete all CRDs:
 
     {{< warning >}}
-    Deleting CRDs will delete any configuration changes that you have made to Istio.
+    Deleting CRDs deletes any configuration changes that you have made to Istio.
```

Comment:
Love the change to present tense! I would move the warning before the step though.